projects/kit/components/accordion/accordion-item/accordion-item.template.html

```diff
@@ -28,12 +28,16 @@
         ></tui-svg>
     </div>
     <tui-expand [async]="async" [expanded]="open">
+        <div *ngIf="eagerContent" class="content">
+            <ng-content select="[tuiAccordionItemContent]"></ng-content>
+        </div>
         <ng-template tuiExpandContent>
             <div
+                *ngIf="lazyContent"
                 polymorpheus-outlet
                 automation-id="tui-accordion__item-content"
                 class="content"
-                [content]="content"
+                [content]="lazyContent"
             ></div>
         </ng-template>
```

Comment:
I introduced `ngIf` because `ng-template` is always projected in the `expand` component therefore we ended up with duplicated styles from the `.content` class. 

I'm not sure if it is a correct implementation

Reply:
I don't see a better solution so let's keep it that way 👍